Feature/Assemble Withdraw Review Page DAO-592

RakeshUP opened the pull request.

Fabricevladimir commented on packages/web-app/src/containers/reviewWithdraw/index.tsx:
```diff
@@ -0,0 +1,140 @@
+            tokenImageUrl={values.tokenImgUrl}
+            tokenSymbol={values.tokenSymbol}
+            tokenCount={values.amount}
+            treasuryShare="$TODO:"
```
This is the token price correct? We fetch this back in the token menu, we can either add it to the form, or fetch it again here considering that the user might take eternity before they complete the form. Thoughts? 

Also, there is a fetchTokenPrice function that takes the token address if you want to use that. Currently it's only supporting Ethereum but we'll modify it once the Apollo client cacheing is finalized.

RakeshUP replied:
formMethod setValues is not re-rendering the component. So if we fetch the token price on the review withdrawal screen, we need to add a state here, which seems too much. Let's see if someone complains that the price is not updated!

Fabricevladimir reviewed (commented):
Good work! Thanks.

Fabricevladimir commented:
One thing I also noticed is that because the back and continue buttons are outside the individual steps, on the reviewProposal, the Publish Proposal button is off since that page takes up the full width of the container. Should a custom footer be passed into the step instead? Thoughts? 🤔

WissenIstNacht commented on packages/web-app/src/locales/en/common.json:
```diff
@@ -245,8 +245,7 @@
       "description": "For a withdraw, a vote is necessary so that the majority can determine whether your proposal is legitimate. For this, you need this configuration.",
       "optionLabel": {
-        "title": "Options",
-        "helptext": "This is a placeholder"
```
Thanks for reminding me of this. I've dropped a question on figma now.

RakeshUP commented:
@Fabricevladimir I've made the requested changes. Regarding the footer for fullWidth I too think having a custom footer prop would be a nice way. Let's see what @sepehr2github does in his create DAO review page PR!

@WissenIstNacht I've addresses your comments too

RakeshUP commented:
@WissenIstNacht what did you put as url for links to open as relative paths? I am not able to reproduce the bug. 
Could you also check the behavior in the deployment preview?

Fabricevladimir commented:
Thanks! It would be nice if you could create a ticket for the link fix. Because abc.com is considered as a valid email, it's appended to the current location. This one is on me 😅  The regular expression needs to require a protocol or we can append one if it's not present.
